test/text_overlay/text_overlay_native_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule TextOverlay.NativeTest do
+  use ExUnit.Case, async: true
+  alias Membrane.FFmpeg.VideoFilter.TextOverlay.Native
+  alias TextOverlay.Helpers
+
+  test "overlay text over raw video frame" do
+    {in_path, out_path, ref_path} = Helpers.prepare_paths("1frame.yuv", "native", ".yuv")
```

Comment:
This test can only be async if you make sure nothing interferes with files and paths. To ensure `test_name` is unique maybe you could use `__MODULE__` as part of the name?

Reply:
the current helper is specifically for TextOverlay element, and it creates all paths inside text_overlay folder (when new elements are added the folder can be parametrized), so names repeating in different modules shouldn't be an issue - aside from module and it's native counterpart.  
Is this solution ok? (https://github.com/membraneframework/membrane_ffmpeg_video_filter_plugin/blob/develop/test/support/text_overlay_helpers.ex#L7)

Reply:
also, is it okay in async test to be using the same file for read in multiple files? (as file 640x360.h264 is used)

Reply:
1. Imagine someone else (or you in 6 months) finds a bug and adds a test by copying the existing one. The solution won't be enough. Also, the helper could be generic enough to be used in other tests. This is of course nitpicking as your solution is OK for now. EDIT: I think I've found the best way: https://hexdocs.pm/ex_unit/1.12/ExUnit.Case.html#module-tmp-dir
2. I think so, concurrent read should be OK (it's handled at OS level and IIRC open with the read-only flag should allow that)